crawl-ref/source/dat/des/branches/swamp.des

```diff
@@ -1260,6 +1285,175 @@ tt... PppPqqq    QqqQQqQqqwwwwww.t
            PPPP    PPPP       ttt
 ENDMAP
 
+# The various "reclaimed" decor vaults just place a pattern of rock walls over
+# the terrain, as if the swamps and forest had formed around the ruins.
+# Sometimes they end up barely visible, enclosed entirely in mangroves except
+# for a corner or something. This is acceptable.
+NAME:  nicolae_swamp_reclaimed_barracks
```

Comment:
I think I mentioned this in discord but it would be cool to get a pattern like this working in orc with crystal walls.